kinder/pkg/kubeadm/automaticcopycertspatch.go

```diff
@@ -65,3 +66,16 @@ metadata:
   name: config
 controlPlane:
   certificateKey: "%s"`
+
+const automaticCopyCertsInitv1beta3 = `apiVersion: kubeadm.k8s.io/v1beta3
+  kind: InitConfiguration
+  metadata:
+	name: config
```

Comment:
this patch is failing

maybe `name: config` should be indented only with 2 spaces.